crates/fj-viewer/src/input/handler.rs

```diff
@@ -146,7 +146,12 @@ impl Handler {
         self.zoom.discard_old_events(now);
         self.zoom.update_speed(now, delta_t, focus_point, camera);
 
-        camera.translation.z -= self.zoom.speed();
+        camera.translation = camera.translation
+            * Transform::translation(Vector::from_components_f64([
+                0.0,
+                0.0,
+                self.zoom.speed(),
+            ]));
```

Comment:
This inverts the mouse wheel behavior. Previously, the metaphor was "mouse wheel moves model", this switches it to "mouse wheel moves camera". Also, the `Vector` construction can be shortened to `Vector::from`

This should do it, I think:
```suggestion
        camera.translation = camera.translation
            * Transform::translation(Vector::from([
                0.0,
                0.0,
                -self.zoom.speed(),
            ]));
```